MIGRATION.md

```diff
@@ -663,83 +630,88 @@ export default {
+```js
+import { app } from '@storybook/vue3';
+import Button from './Button.vue';
 
-#### Icons API changed
+app.component('GlobalButton', Button);
+```
 
-For addon authors who use the `Icons` component, its API has been updated in Storybook 7.
+After:
 
-```diff
-export interface IconsProps extends ComponentProps<typeof Svg> {
--  icon?: IconKey;
--  symbol?: IconKey;
-+  icon: IconType;
-+  useSymbol?: boolean;
-}
+```js
+import { setup } from '@storybook/vue3';
+import Button from './Button.vue';
+
+setup((app) => {
+  app.component('GlobalButton', Button);
+});
 ```
 
-Full change here: https://github.com/storybookjs/storybook/pull/18809
+### Addon authors
 
-#### 'config' preset entry replaced with 'previewAnnotations'
+#### register.js removed
 
-The preset field `'config'` has been replaced with `'previewAnnotations'`. `'config'` is now deprecated and will be removed in Storybook 8.0.
+In SB 6.x and earlier, addons exported a `register.js` entry point by convention, and users would import this in `.storybook/manager.js`. This was [deprecated in SB 6.5](#deprecated-registerjs)
 
-Additionally, the internal field `'previewEntries'` has been removed. If you need a preview entry, just use a `'previewAnnotations'` file and don't export anything.
+In 7.0, most of Storybook's addons now export a `manager.js` entry point, which is automatically registered in Storybook's manager when the addon is listed in `.storybook/main.js`'s `addons` field.
 
-#### Dropped support for Angular 12 and below
+If your `.manager.js` config references `register.js` of any of the following addons, you can remove it: `a11y`, `actions`, `backgrounds`, `controls`, `interactions`, `jest`, `links`, `measure`, `outline`, `toolbars`, `viewport`.
 
-Official [Angular 12 LTS support ends Nov 2022](https://angular.io/guide/releases#actively-supported-versions). With that, Storybook also drops its support
-for Angular 12 and below.
+#### No more default export from `@storybook/addons`
 
-In order to use Storybook 7.0, you need to upgrade to at least Angular 13.
+The default export from `@storybook/addons` has been removed. Please use the named exports instead:
 
-#### Addon-backgrounds: Removed deprecated grid parameter
+```js
+import { addons } from '@storybook/addons';
+```
 
-Starting in 7.0 the `grid.cellSize` parameter should now be `backgrounds.grid.cellSize`. This was [deprecated in SB 6.1](#deprecated-grid-parameter).
+The named export has been available since 6.0 or earlier, so your updated code will be backwards-compatible with older versions of Storybook.
 
-#### Addon-docs: Removed deprecated blocks.js entry
+#### No more configuration for manager
 
-Removed `@storybook/addon-docs/blocks` entry. Import directly from `@storybook/blocks` instead. This was [deprecated in SB 6.3](#deprecated-scoped-blocks-imports).
+The storybook manager is no longer built with webpack. Now it's built with esbuild.
```

Comment:
@IanVS one more capitalization here it should be Storybook and Webpack if you wouldn't mind.